CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 add_subdirectory(common)
 add_subdirectory(scan)
 add_subdirectory(constant)
-add_subdirectory(radix)
+add_subdirectory(sort)
```

Comment:
you also left radix library name below.

Reply:
I thought it would be more correct to combine tbb sort and radix sort into one directory where the sorts will be located
Is it better to separate them?

Reply:
No, that's fine too. Probably rename the sort one to a ttb sort?